src/styles/public/navigation.scss

```diff
@@ -246,6 +246,7 @@ body.has-ribbon-menu {
 		&__container {
 			flex-direction: row !important;
```

Comment:
Just curious about the use of `!important` here. Are you fighting against some other inaccessible styles?